apparmor.d/groups/lxqt/featherpad

```diff
@@ -0,0 +1,57 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+#include <tunables/global>
+
+@{exec_path} = @{bin}/featherpad
+profile featherpad @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/common/app>
```

Comment:
This abstraction must not be used for this kind of profile. See https://apparmor.pujol.io/development/abstractions/#commonapp